tsdb_k8s_queries/container-template.json

```diff
@@ -107,6 +115,9 @@
             "name": "kubernetes"
           }
         },
+        "_source": {
```

Comment:
Not critical, but when is index.mode=time_series, then source mode is always synthetic. Do you know where this comes from? Ideally this shouldn't be defined.

I think the solutions should rely the default that is derived from `index.mode` index setting, which determined in Elasticsearch.